src/actions/index.js

```diff
@@ -44,9 +44,16 @@ export const createList = (boardId, listName) => {
 	}
 
 	const parsed = JSON.parse(localStorage.getItem("listCollection"));
+	const ids = () => {
+		if (parsed.length === 1) {
+			return 1;
```

Comment:
I dont see the use case of this if? if 1==1 return 1? why dont you just return parsed.length?